src/modules/roc_packet/fec.h

```diff
@@ -21,11 +21,11 @@ namespace packet {
 
 //! FECFRAME packet.
 struct FEC {
-    //! Seqnum of first source packet in block.
-    seqnum_t blknum;
+    //! The source block number.
+    blknum_t source_block_number;
```

Comment:
It's worth to emphasize in the comment that:

* this is the number of a block, not a packet
* blocks are numbered sequentially starting from a random number
* block number can wrap

Reply:
Done